src/sim_engine/pool.rs

```diff
@@ -2,18 +2,24 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+use std::collections::BTreeMap;
 use std::vec::Vec;
 
 use engine::EngineResult;
 use engine::Pool;
 
 use super::blockdev::SimDev;
 
+#[derive(Debug, Clone)]
```

Comment:
Maybe you could break this currently tiny definition and implementation out to a separate file. It will get bigger.